src/match_string.h

```diff
@@ -15,6 +13,38 @@
  * index of a @string in the @array if matches, or %-EINVAL otherwise.
  */
 
+/*@
+    logic integer cmp(unsigned char a, unsigned char b) =
+       a == b ? 0 : a < b ? -1 : 1;
+
+    logic integer strncmp(char *cs, char *ct, integer n) =
+       n == -1 ? 0 : (cs[n] == ct[n] ? strncmp(cs+1, ct+1, n-1) : cmp((u8 AENO)cs[n], (u8 AENO)ct[n]));
+    logic integer strcmp(char *cs, char *ct) = strncmp(cs, ct, strlen(cs));
+*/
```

Comment:
These definitions should be included from strcmp.h file. Please, don't copy-paste them here.

Reply:
There is no @ symbol in comments in strcmp.h line 6. This causes an error "unbound function strcmp.". Should I add '@'?

Reply:
Yes, you can edit these logic functions in strcmp.h